tutorials/using-cloud-vpn-with-cisco-asr/index.md

```diff
@@ -0,0 +1,899 @@
+
+!
+
+interface <b>Tunnel2</b>
+
+description VPN tunnel to the west coast DC
+
+ ip address <b>169.254.0.6 255.255.255.252</b>
+
+ ip mtu 1400
+
+ ip tcp adjust-mss 1360
+
+ tunnel source <b>TenGigabitEthernet0/0/0</b>
+
+ tunnel mode ipsec ipv4
+
+ tunnel destination <b>35.186.108.199</b>
+
+ tunnel protection ipsec profile <b>VPN_SCALE_TEST_VTI_2</b>
+
+!
+</pre>
+
+Dynamic Routing
+
+<pre>
+router bgp <b>65001</b>
+
+ bgp log-neighbor-changes
+
+ neighbor <b>169.254.0.1</b> description BGP session over Tunnel1
+
+ neighbor <b>169.254.0.1</b> remote-as <b>65002</b>
+
+ neighbor <b>169.254.0.1</b> timers <b>20 60 60</b>
+
+ neighbor <b>169.254.0.5</b> description BGP session over Tunnel2
+
+ neighbor <b>169.254.0.5</b> remote-as <b>65002</b>
+
+ neighbor <b>169.254.0.5</b> timers <b>20 60 60</b>
+
+ !
+
+ address-family ipv4
+
+  network <b>10.0.0.0</b>
+
+  neighbor <b>169.254.0.1</b> activate
+
+  neighbor <b>169.254.0.1</b> route-map LP200 in
+
+   neighbor <b>169.254.0.5</b> activate
+
+ exit-address-family
+
+!
+
+route-map LP2000 permit 10
+
+ set local-preference 2000
+
+</pre>
+
+To ensure symmetry in your traffic flow, you can configure MED to influence the inbound traffic from GCP for the same tunnel you are sending outbound traffic to. Note that lower the MED, higher the preference.
+
+<pre>
+
+router bgp <b>65001</b>
+
+ address-family ipv4
+
+  neighbor <b>169.254.0.1</b> route-map SET-MED-10 out
+
+   neighbor <b>169.254.0.5</b> activate
+
+ exit-address-family
+
+!
+
+route-map SET-MED-10 permit 10
+
+ set metric 10
+
+</pre>
+
+Static Routing
+
+If you are using static routing then instead of BGP configurations mentioned above, you can change the metric (higher the metric lower the preference) for your static route as shown below:
+
+
+```
+cisco-asr#ip route 172.16.100.0 255.255.255.0 Tunnel2 10
+```
+
+#### Configuring Route Priority – Google Cloud Platform (GCP)
+
+Dynamic Routing (Optional)
+
+With dynamic routing you have an option to define advertised-route-priority, lower the priority preferred it is. More details can be found [here](https://cloud.google.com/sdk/gcloud/reference/compute/routers/update-bgp-peer). Note that if you have local_preference configured on the peer network as mentioned [above](#bookmark=id.rkss8sm16r9x), BGP will prefer the higher local_preference first.
+
+
+```
+gcloud compute --project vpn-guide routers add-bgp-peer vpn-scale-test-cisco-rtr --peer-name bgp-peer1 --interface if-1 --peer-ip-address 169.254.1.2 --peer-asn 65001 --region us-east1 --advertised-route-priority=2000
+```
+
+
+Static Routing (Optional)
+
+When using static routing GCP provides you an option to customize the priority in case there are multiple routes with the same prefix length . In order to have symmetric traffic flow make sure that you set the priority of your secondary tunnel to higher value than the primary tunnel (default priority is 1000). To define the route priority run the below command.
```

Comment:
There's a space before the period after "length".

Reply:
Done